packages/server-core/src/media/file-browser/file-browser.class.ts

```diff
@@ -157,7 +157,7 @@ export class FileBrowserService
       const allowedProjectNames = projectPermissions.map((permission) => permission.project.name)
       result = result.filter((item) => {
         const projectRegexExec = /projects\/(.+)$/.exec(item.key)
-        const subFileRegexExec = /projects\/(.+)\//.exec(item.key)
+        const subFileRegexExec = /projects\/([^\/]+\/[^\/]+)/.exec(item.key)
```

Comment:
@hanzlamateen I changed this regex so that it captures the project for folder, it only captured the project for files, so now sub-folders in a project are also displayed.

Reply:
I hope this will work for both of following:

1. /projects/project1
2. /projects/account/project1

Because example 1 represents how things work in standalone core engine and example 2 represents how things work in engine with multitenancy.

Reply:
It does as long as the user has a project permission for that project. 
This section of the code is executed only when users don't have admin scope Files & folders are filtered by project permission. 

Let me know If we need to change that too.

Reply:
You're right. Initially the regex only captured the project for projects with the studio running on standalone
my change only captured, I updated the filter and made it simpler.